prefect_hightouch/syncs/flows.py

```diff
@@ -0,0 +1,199 @@
+"""
+This is a module containing flows used for interacting with syncs.
+"""
+
+import asyncio
+from typing import Tuple
+
+from prefect import flow, get_run_logger
+
+from prefect_hightouch.api_client.models import Sync, SyncStatus, TriggerRunInput
+from prefect_hightouch.credentials.generated import HightouchCredentials
+from prefect_hightouch.exceptions import (
+    TERMINAL_STATUS_EXCEPTIONS,
+    HightouchSyncRunTimedOut,
+)
+from prefect_hightouch.syncs import get_sync, trigger_run
+
+
+@flow
+async def trigger_sync_run_and_wait_for_completion(
+    hightouch_credentials: HightouchCredentials,
+    sync_id: str,
+    full_resync: bool = True,
```

Comment:
The API defaults this to false, so I think we should too.
```suggestion
    full_resync: bool = False,
```